application/views/admin/token/csvimportresult.php

```diff
@@ -101,6 +102,24 @@
                                 </li>
                             <?php } ?>
 
+                            <?php if (!empty($aPluginErrorMessageList)) { 
+                                        $iPluginErrorIndex=0;
+                                        foreach ($aPluginErrorMessageList as $sPluginErrorMessage => $aTokenSpecificErrorList) { 
```

Comment:
Consider using alternative syntax here.

https://www.php.net/manual/en/control-structures.alternative-syntax.php

Reply:
At this line ? I didn't see how :)

Reply:
On all lines.

Well, it gets hard because of this: "$iPluginErrorIndex=0;". But lines like that don't belong in viewfiles. Hence, widget might be better?

Reply:
Widget for this would be an overkill I think.

> But lines like that don't belong in viewfiles. 
Don't agree too much. I understand alternative syntax is better for view files, but this is just used for numbering. This kind of logic doesn;t belong to a controller either. I would say is more a view logic.

I would just leave it like that or maybe just chnage it to the Alternative Syntax using multiple lines.
But honestyl, I don't believe it worth doing it.

Reply:
I don t see any needs to use alternative synthax. is that needed @olleharstedt ?

Reply:
Me and Louis used alternative syntax as we thought it had higher readability. But it's not mandated in any style guide.